scripts/jrelease.java

```diff
@@ -0,0 +1,347 @@
+/*
+ * Copyright (c) 2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+///usr/bin/env jbang "$0" "$@" ; exit $?
+// //DEPS <dependency1> <dependency2>
+//DEPS org.jline:jline:3.16.0
+//DEPS de.codeshelf.consoleui:consoleui:0.0.13
+//DEPS org.fusesource.jansi:jansi:2.3.4
+//DEPS info.picocli:picocli:4.6.1
+//DEPS org.apache.maven:maven-model:3.8.1
```

Comment:
These might need an eclipse-dash check.

- Run the `eclipse-dash-tool.sh script.
- Then add these dependencies manually to the target/dependencies.txt list (following the same pattern.
- Then execute the last part of the script manually: 
  ```shell
  java -jar target/eclipse-dash.jar target/dependencies.txt -summary  target/dependencies-resolved.csv
  ```

Reply:
```
$ (jrelease) java -jar target/eclipse-dash.jar target/dependencies.txt -summary  target/dependencies-resolved.csv
[main] INFO Querying Eclipse Foundation for license data for 264 items.
[main] INFO Found 159 items.
[main] INFO Querying ClearlyDefined for license data for 105 items.
[main] INFO Found 105 items.
[main] INFO License information could not be automatically verified for the following content:
[main] INFO 
[main] INFO maven/mavencentral/de.codeshelf.consoleui/consoleui/0.0.13
[main] INFO 
[main] INFO This content is either not correctly mapped by the system, or requires review.
```

Reply:
I just harvested it, try to repeat and see if the java -jar command and see if it's OK now.

https://clearlydefined.io/definitions/maven/mavencentral/de.codeshelf.consoleui/consoleui/0.0.13

It has a score of 60, I can't remember if this was enough.